### PR TITLE
METRON-102: Unit Tests for parsers and enrichment adapters

### PR DESCRIPTION
**Pull request by merrimanr**
Added several new unit tests for existing parsers and enrichment adapters.  One of the tests needed the MockHTable class which currently lives in Metron-Testing.  Since Metron-Testing depends on several other projects (including the project that needs MockHTable), a separate Metron-TestingUtilities project was created with no dependencies on any projects.  Now tests written for any project have access to this class.

### CODE REVIEW

File: metron-streaming/pom.xml [hunk 1]
```diff
@@ -80,6 +80,7 @@
 		<module>Metron-Elasticsearch</module>
 		<module>Metron-Solr</module>
 		<module>Metron-Testing</module>
+		<module>Metron-TestingUtilities</module>
```
Comment (dlyle65535) [inline metron-streaming/pom.xml]: What's the difference between TestingUtilities and Testing?

Comment (merrimanr) [inline metron-streaming/pom.xml]: Testing contains our integration test classes and depends on pretty much every project (Common, Parsers, etc).  TestingUtilities is meant to contain testing utilities (like MockHTable) that any project can leverage.

**Comment by dlyle65535** [comment]
+1 this looks really good. Ran through the tests no problem.

**Comment by merrimanr** [comment]
@franklu1 put in a lot of work for this PR but his name was in a commit that got squashed.  Whoever merges this please include him so he gets credit.

**Pull request closed without merging**